Purchases/Identity/CustomerInfoManager.swift

```diff
@@ -0,0 +1,185 @@
+//
```

Comment:
It wouldn't let me `git mv` this file for some reason, each time I did it, it still tracked it as a removal of the original and addition of this.